packages/server/src/unstable-core-do-not-import/http/resolveResponse.ts

```diff
@@ -504,7 +504,7 @@ export async function resolveResponse<TRouter extends AnyRouter>(
     // batch response handlers
     if (info.accept === 'application/jsonl') {
       // httpBatchStreamLink
-      headers.set('content-type', 'application/json');
+      headers.set('content-type', 'text/event-stream+json');
```

Comment:
Should it be `application/stream+json`?

Was just trying to look it up out of interest and only found this:

https://stackoverflow.com/questions/52098863/whats-the-difference-between-text-event-stream-and-application-streamjson

Reply:
Nice. Going to try these and see if the network tab in chrome displays them nicely

Reply:
None of `application/*` looks nice in the output in chrome inspector

Might look at refactoring so it actually gives back an `text/event-stream` because it looks good 😅

Reply:
`text/event-stream` seems to be the one that gives the nicest output


![CleanShot 2025-06-28 at 16 16 46@2x](https://github.com/user-attachments/assets/f8a60ab8-13d2-4d7d-9895-3bb5dbe516c4)

Reply:
but i won't care about that for now